File: tools/elevenlabs-isolate-voice/tool.test.ts
import { run } from './tool.ts';
const ELEVENLABS_API_KEY = 'sk_live_11labs_api_key';

const config = {
  ELEVENLABS_API_KEY: ELEVENLABS_API_KEY
}

const inputs = {
  youtube_url: 'https://www.youtube.com/watch?v=ChQDGRH7OSY'
}

const result = await run(config, inputs);
console.log(result);

File: tools/audio-to-text/tool.test.py
# /// script
# requires-python = ">=3.10,<3.12"
# dependencies = [
#   "requests",
#   "faster-whisper",
#   "asyncio",
# ]
# ///
import asyncio
from tool import CONFIG, INPUTS, run


def test_basic():
    config = CONFIG()
    inputs = INPUTS()
    inputs.audio_file_path = "tests_data_physicsworks.wav"
    result = asyncio.run(run(config, inputs))
    print(result.transcript)

test_basic()